addons/html_builder/static/src/core/builder_action.js

```diff
@@ -61,6 +62,7 @@ export class BuilderAction {
     /**
      * Clean/reset the value if needed.
      * @param {Object} context
+     * @param {boolean} context.isPreviewing
```

Comment:
Not link to your PR, could you fix the doc string we forget:
     * @param {any} context.value
     * @param {Object} [context.params]